### PR TITLE
Update IBC version to v1.0.0-beta1

### LINKED ISSUE #23: Update IBC version to v1.0.0-beta1

### PR DESCRIPTION
**Pull request by bluele**
fixes #23

### CODE REVIEW

File: tests/chains/tendermint/simapp/test_helpers.go [hunk 1]
```diff
@@ -442,3 +337,13 @@ type EmptyAppOptions struct{}
 func (ao EmptyAppOptions) Get(o string) interface{} {
 	return nil
 }
+
+// FundAccount is a utility function that funds an account by minting and sending the coins to the address
+// TODO(fdymylja): instead of using the mint module account, which has the permission of minting, create a "faucet" account
+func FundAccount(app *SimApp, ctx sdk.Context, addr sdk.AccAddress, amounts sdk.Coins) error {
```
Comment (siburu) [inline tests/chains/tendermint/simapp/test_helpers.go:343]: Do we need this function? Even if so, the copied todo comment should be removed.

File: chains/tendermint/chain.go [hunk 2]
```diff
@@ -253,23 +265,124 @@ func (c *Chain) sendMsgs(msgs []sdk.Msg) (res *sdk.TxResponse, err error) {
...
+		return nil, fmt.Errorf("cannot simulate amino tx")
+	}
+	simReq := txtypes.SimulateRequest{Tx: protoProvider.GetProtoTx()}
+
+	return simReq.Marshal()
+}
+
+// CalculateGas simulates the execution of a transaction and returns the
+// simulation response obtained by the query and the adjusted gas amount.
+func CalculateGas(
```
Comment (siburu) [inline chains/tendermint/chain.go:366]: I understand that these codes have been temporarily copied and will be removed later, is that correct?

Comment (bluele) [reply chains/tendermint/chain.go:366]: I think It's probably correct.
This change is based on the following: https://github.com/cosmos/relayer/pull/501/files#diff-ba13e4822ca5775807a70be387519276a6103762c525afb11068bdff525d7fcbR299
For now, I'd like to use cosmos/relayer codebase as much as possible to reduce the maintenance cost of tendermint support.

Comment (siburu) [reply chains/tendermint/chain.go:366]: :+1:

File: tests/chains/tendermint/simapp/test_helpers.go (hunk 1 above)
Comment (siburu) [reply tests/chains/tendermint/simapp/test_helpers.go:343]: OK. This entire file was intentionally copied, so we should not edit this file.

**Review by siburu — APPROVED** [review]
:100:

**Pull request merged**